interfaces/tls_certificates/v1/README.md

```diff
@@ -2,13 +2,15 @@
 
 ## Usage
 
-This relation interface describes the expected behavior of any charm claiming to be able to provide 
+This relation interface describes the expected behavior of any charm claiming to be able to provide
 or consume TLS certificates.
 
+A recommended approach is to use the [tls_certificates_interface library V4](https://github.com/canonical/tls-certificates-interface/blob/main/lib/charms/tls_certificates_interface/v4/tls_certificates.py).
```

Comment:
I'm curious (mostly because of other discussions that are happening) about the choice to link to the code in GitHub and not to CharmHub. What's the reasoning for that?

cc @james-garner-canonical

Reply:
I don't have a strong opinion here. The code on Github seems more explicit since Charmhub doesn't list all versions of the lib there.
I could link to [this](https://charmhub.io/tls-certificates-interface/docs/h-getting-started) tutorial on Charmhub too in case of V4.